app/lib/constants/colors.ts

```diff
@@ -1,3 +1,5 @@
+export const BACKGROUND_PUSH_COLOR = '#F5455C';
```

Comment:
I didn't see this on screenshots. Why is it red? 😅

Reply:
It's the official color of rocket chat kek